src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -2544,10 +2544,18 @@ internal static string GetInterpolatedStringCustomHandlerType(string name, strin
             var cultureInfoHandler = @"
 public class CultureInfoNormalizer
 {
+    private static CultureInfo originalCulture;
+
     public static void Normalize()
```

Comment:
>static [](http://example.com/codeflow?start=11&length=6)

Consider having `Normalize()` return an `IDisposable` to allow callers to use `using`.

Reply:
> > static
> 
> Consider having `Normalize()` return an `IDisposable` to allow callers to use `using`.

I intentionally did not do this to minimize change to existing test IL verification.